A4_3.java

```diff
@@ -0,0 +1,34 @@
+import java.util.Scanner;
+
+int+push;

```

Comment:
What does this do? Did you forget to remove it?

Reply:
yes, I forgot. Merci!
I got lots of spaces and empty lines because the teacher is asking to space out the lines.